platform-server-rest/src/main/java/uno/cod/platform/server/rest/RestUrls.java

```diff
@@ -20,6 +20,8 @@
     public static final String TASKS = "/tasks";
     public static final String TASKS_ID = "/tasks/{id}";
 
+    public static final String RESULTS_TASKS_SUBMISSIONS = "/results/{resultId}/tasks/{taskId}/submissions";
```

Comment:
That's fine. A result may be computed from all (or only selected) Submissions. A Result links a User to a Challenge, and we relate the uploads (read: Submissions) to it.